Makefile

```diff
@@ -1,5 +1,6 @@
 NAME       := mercury
-SRC_EXT    := bz2
+SRC_EXT    := gz
+DL_VERSION := 2.0.1rc1
```

Comment:
Make the following change in packaging/Makefile_packaging.mk:

```
--- a/packaging/Makefile_packaging.mk
+++ b/packaging/Makefile_packaging.mk
@@ -131,7 +131,7 @@ _topdir/SOURCES/%: % | _topdir/SOURCES/
 # At least one spec file, SLURM (sles), has a different version for the
 # download file than the version in the spec file.
 ifeq ($(DL_VERSION),)
-DL_VERSION = $(VERSION)
+DL_VERSION = $(subst ~,,$(VERSION))
 endif
 ifeq ($(DL_NAME),)
 DL_NAME = $(NAME)
```

and you can get rid of the above line.